src/main/java/teammates/ui/template/ElementTag.java

```diff
@@ -32,6 +32,10 @@ public ElementTag(String... attributePairs) {
         }
     }
     
+    public void setContent(String contentParam) {
+        content = contentParam;
+    }
+    
```

Comment:
Do use `this.content = content` here instead, following the style of this file.
Personally, I would recommend not having this method at all; you can set the content with the initial call to the constructor and then the attributes through `setAttribute()`.